plugins/LilyPad/LilyPad.rc

```diff
@@ -138,6 +136,78 @@ BEGIN
     PUSHBUTTON      "Small Motor",ID_SMALL_MOTOR,347,291,64,14
 END
 
+IDD_CONFIG_POPN DIALOGEX 0, 0, 424, 318
+STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | WS_POPUP | WS_CAPTION
+FONT 8, "MS Shell Dlg", 0, 0, 0x1
+BEGIN
+    CONTROL         "",IDC_LIST,"SysListView32",LVS_REPORT | LVS_SHOWSELALWAYS | LVS_NOSORTHEADER | WS_BORDER | WS_VSCROLL | WS_TABSTOP,7,7,183,285,WS_EX_CLIENTEDGE
+    PUSHBUTTON      "Delete Selected",ID_DELETE,7,296,59,15
+    PUSHBUTTON      "Clear All",ID_CLEAR,71,296,56,15
+    PUSHBUTTON      "Ignore Key",ID_IGNORE,132,296,58,15
+	
+	PUSHBUTTON      "Select", ID_SELECT, 262, 22, 43, 15
+	PUSHBUTTON      "Start", ID_START, 306, 22, 43, 15
+
+    PUSHBUTTON      "White L",ID_TRIANGLE, 196,66,43,15
```

Comment:
lines around 147-150 needs proper alignment. also don't leave a space after the elements at comma for all the Resource script codes.

Example:
``"Start", ID_START, 306, 22, 43, 15`` should be changed to ``"Start",ID_START,306,22,43,15"``